code/src/App.js

```diff
@@ -1,9 +1,23 @@
-import React from 'react';
+// src/App.js
+import React from 'react'
+import { Provider } from 'react-redux'
+import { combineReducers, configureStore } from '@reduxjs/toolkit'
+import Tasks from 'components/TaskList'
+import tasks from 'reducers/tasks'
+import Header from 'components/Header'
+
+const reducer = combineReducers({
+  tasks: tasks.reducer
+})
+
+const store = configureStore({ reducer })
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  );
-}
+    <Provider store={store}>
+      <Header />
+      <tasks />
```

Comment:
I'm not sure u need the reducer here. I don't know what it does or if it actually displays something.